app/src/main/java/com/sopt/dive/util/MyApplication.kt

```diff
@@ -0,0 +1,15 @@
+package com.sopt.dive.util
+
+import android.app.Application
+
+
+class MyApplication : Application() {
+    companion object {
+        lateinit var prefs: PreferenceUtil
```

Comment:
```suggestion
        lateinit var prefs: PreferenceUtil
            private set
```
이렇게 private set을 지정해 외부에서 재할당되지 않도록 하면 더 안전합니다